Dockerfile.sget

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 The Sigstore Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG RUNTIME_IMAGE=gcr.io/distroless/base:debug
```

Comment:
this is only going to be compatible with x86(-64)

Reply:
For the release we ise this gcr.io/distroless/static:debug-nonroot

This also is only for x86?  

I built locally and that built for arm.

Vut can we replace that or should we drop the arm for container?


Thanks for the review!

Reply:
You're right, I forgot we made the arch-y images multi-arch